src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionDownloader.java

```diff
@@ -31,10 +33,16 @@ public class TranscriptionDownloader {
     private final AuditApi auditApi;
     private final UserIdentity userIdentity;
 
-
     public DownloadTranscriptResponse downloadTranscript(Integer transcriptionId) {
         var userAccountEntity = getUserAccount();
-        var transcriptionEntity = transcriptionRepository.findById(transcriptionId).orElseThrow(() -> new DartsApiException(TRANSCRIPTION_NOT_FOUND));
+        var userIsSuperAdmin = this.userIdentity.userHasGlobalAccess(Set.of(SUPER_ADMIN));
+        var transcriptionEntity = transcriptionRepository.findById(transcriptionId)
+            // Only SUPER_ADMIN users are allowed to download hidden documents
+            .filter(transcription -> userIsSuperAdmin
+                || transcription.getTranscriptionDocumentEntities().isEmpty()
+                || transcription.getTranscriptionDocumentEntities().stream().noneMatch(TranscriptionDocumentEntity::isHidden)
```

Comment:
Do we want to exclude the entire transcription or just remove the hidden transcription document from the response?

Reply:
Yes we do. When downloading the document for a transcription we currently assume there is only one and if there are multiple we send the most recently uploaded one.

Reply:
So what would happen if the most recent is not hidden but a less recent one is?

Reply:
If any are hidden, we assume they are all hidden. All the code operates as if there can only be 1 document, but the data model allows multiple so this is just a belt and braces check really.